.bintray

```diff
@@ -1,2 +1,2 @@
-BINTRAY_PROJECT=shippedrepos/mantl-1.2
+BINTRAY_PROJECT=shippedrepos/mantl-1.2-k8s
```

Comment:
this will change the bintray project for all the packages. Is that what you want?